contentcuration/contentcuration/tests/viewsets/test_contentnode.py

```diff
@@ -681,6 +681,22 @@ def test_update_contentnode_tags(self):
             .exists()
         )
 
+    def test_update_contentnode_suggested_duration(self):
```

Comment:
Could you add a parallel test in the `CRUDTestCase` to check that this all works for RESTful operations as well?